sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -0,0 +1,682 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+import pytest
+import platform
+import functools
+import itertools
+
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
+from azure.core.credentials import AzureKeyCredential
+from testcase import TextAnalyticsTest, GlobalTextAnalyticsAccountPreparer
+from testcase import TextAnalyticsClientPreparer as _TextAnalyticsClientPreparer
+from azure.ai.textanalytics import (
+    TextAnalyticsClient,
+    TextDocumentInput,
+    VERSION,
+    TextAnalyticsApiVersion,
+)
+
+# pre-apply the client_cls positional argument so it needn't be explicitly passed below
+TextAnalyticsClientPreparer = functools.partial(_TextAnalyticsClientPreparer, TextAnalyticsClient)
+
+class TestHealth(TextAnalyticsTest):
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net"
+    })
```

Comment:
could this be somehow extracted into a function so that you don't have to initialize the client for every test?

Reply:
I actually tried to move this to a variable but it didn't work for some reason.  It seemed to work for the first test that ran, but all others would use default settings which pointed to the wrong endpoint.  I will keep this open for now to see if I can come up with another solution for this.